pkg/prometheus/server/statefulset_test.go

```diff
@@ -2237,6 +2237,43 @@ func TestPrometheusAdditionalArgsDuplicate(t *testing.T) {
 	require.Contains(t, err.Error(), expectedErrorMsg, "expected the following text to be present in the error msg: %s", expectedErrorMsg)
 }
 
+func TestRuntimeGOGCEnvVar(t *testing.T) {
```

Comment:
This test is failing, I am trying to figure out why as when I run the config manually on cluster it works

Reply:
fixed